Makefile

```diff
@@ -39,25 +38,13 @@ logs: ## Shows logs of all running projects' containers
 logs-recent: ## For CI
 	@ make -C docker logs-recent
 
-.PHONY: bash
-bash: ## Starts a bash shell in an app environment
-	@ echo Run this within a specific app context, ie: make -C metis bash
-
-.PHONY: psql
-psql: ## Starts a psql shell in an app environment
-	@ echo Run this within a specific app context, ie: make -C janus psql
-
-.PHONY: migrate
-migrate: ## Runs migrations in a specific app context
-	@ echo Run this within a specific app context, ie: make -C janus migrate
-
 .PHONY: update
 update: ## Runs all projects' updates, running bundle and npm install
 	@ make -C docker update
 
 .PHONY: release
 release: ## Builds static docker images staged for release, runs tests against them, and pushes them to dockerhub (requires PUSH_IMAGES=1)
-	@ set -e && for project in $(projects); do make -C $$project release; done
+	@ make -f Release.mk
```

Comment:
Because the `Release` process involves building all images en masse and can be quite slow, I've pulled this out into a separate makefile.  I may repeat this process in later commits to improve overall make performance.